src/sagas/featureToggle.js

```diff
@@ -144,25 +144,19 @@ export function* monitorFeatureFlags(currentRetry = 0) {
 
 export function* setupUnleashListeners(unleashClient) {
   const channel = eventChannel((emitter) => {
-    const listener = (state) => emitter(state);
+    const l1 = () => emitter({ type: 'FEATURE_TOGGLE_UPDATE' });
+    const l2 = () => emitter({ type: 'FEATURE_TOGGLE_READY' });
+    const l3 = (err) => emitter({ type: 'FEATURE_TOGGLE_ERROR', data: err });
```

Comment:
Is there any specific reason we changed the event listeners from anonymous functions to `const` variables?

Reply:
No specific reason, I had to remove the cleanup below and figured I would create listeners as it seemed like the intention with the old `listener` but I does not have any deep reasoning behind it